test/unit/plugins/providers/docker/action/destroy_network_test.rb

```diff
@@ -3,6 +3,7 @@
 
 describe VagrantPlugins::DockerProvider::Action::DestroyNetwork do
   include_context "unit"
+  include_context "virtualbox"
```

Comment:
This should be removed, along with the other `include_context` lines in this set of tests. They were included by accident originally.